src/FakeItEasy/FakeOptionsExtensions.cs

```diff
@@ -21,12 +21,7 @@ public static IFakeOptions<T> Strict<T>(this IFakeOptions<T> options)
         {
             Guard.AgainstNull(options, nameof(options));
 
-            Action<IFakeObjectCall> thrower = call =>
-                {
-                    throw new ExpectationException("Call to non configured method \"{0}\" of strict fake.".FormatInvariant(call.Method.Name));
-                };
-
-            return options.ConfigureFake(fake => A.CallTo(fake).Invokes(thrower));
+            return options.ConfigureFake(fake => ConfigureStrictFake(fake));

```

Comment:
No, it won't work in this case. (I had written a more complete explanation, but my internet connection crashed and I'm too lazy to type it on my phone...)